app/public/js/ttPerson.js

```diff
@@ -228,6 +235,30 @@ function characterMysteryWalkAnimation() {
     return {frames};
 }
 
+function characterMysteryUncontrolledFallAnimation() {
+    var xSize = 32,
+    ySize = 32,
+    hitBox = new Rectangle(0, -64, 64, 64),
+    frames = [
+        $.extend(new Rectangle(0 * xSize, 0 * ySize, xSize, ySize), {image: fireballBImage, hitBox}),
+        $.extend(new Rectangle(1 * xSize, 0 * ySize, xSize, ySize), {image: fireballBImage, hitBox}),
+        $.extend(new Rectangle(2 * xSize, 0 * ySize, xSize, ySize), {image: fireballBImage, hitBox}),
+        $.extend(new Rectangle(3 * xSize, 0 * ySize, xSize, ySize), {image: fireballBImage, hitBox}),
+        $.extend(new Rectangle(4 * xSize, 0 * ySize, xSize, ySize), {image: fireballBImage, hitBox})
+    ];
+    return {frames};
+}
+
+function characterMysteryJumpAnimation() {
+    var xSize = 32,
+    ySize = 32,
+    hitBox = new Rectangle(-4, -32, 40, 64),
+    frames = [
+        $.extend(new Rectangle(0 * xSize, 0 * ySize, xSize, ySize), {image: characterMysteryImage, hitBox}),
+    ];
+    return {frames};
+}
+
```

Comment:
Locally I was able to replace the walk/idle animations with these functions, in case that helps:
```
function characterMysteryWalkAnimation() {
    frames = rectangleToFrames(new Rectangle(0, 0, 32, 32), characterMysteryImage, 4);
    return {frames};
}

function characterMysteryIdleAnimation() {
    frames = rectangleToFrames(new Rectangle(0, 0, 32, 32), characterMysteryImage, 8).slice(4, 8);
    return {frames};
}
```